_includes/authorizations-resource.md

```diff
@@ -0,0 +1,184 @@
+{% assign payment-instrument = include.payment-instrument | default: creditcard %}
+
+### Authorizations
+
+The `authorizations` resource will list the authorization transactions
+made on a specific payment.
+
+{:.code-header}
+**Request**
+
+```http
+GET /psp/{{payment-instrument}}/payments/<payment-id>/authorizations HTTP/1.1
+Host: api.externalintegration.payex.com
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+```
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "payment": "/psp/{{payment-instrument}}/payments/<payment-id>",
```

Comment:
Please don't encourage URL building in clients; replace `<payment-id>` with a random UUID. You can set that with a Liquid variable, perhaps? Most of the URL can be set in a variable as well, so we avoid repetition in the rest of the file. Thumbs up for parametrized include! 👍